application.rb

```diff
@@ -1,4 +1,8 @@
 require_relative 'product'
+require_relative 'vendor'
 
 # Creating new product
 product = Product.new("Bread", 12)
```

Comment:
Let's use single quotes ' instead of double quotes " when it's not needed.
Also, let's create one more product to see the creation of a vendor with multiple products.